src/rgw/rgw_op.cc

```diff
@@ -6139,6 +6144,81 @@ void RGWCompleteMultipart::execute(optional_yield y)
   }
 }
 
+bool RGWCompleteMultipart::check_previously_completed(const DoutPrefixProvider* dpp, const RGWMPObj& mpobj, const RGWMultiCompleteUpload* parts, rgw::sal::Store* store)
+{
+  // TODO: replace with mpstat() in zipper
+  auto rados = dynamic_cast<rgw::sal::RadosStore*>(store);
```

Comment:
as discussed in the standup, i don't think it's necessary to stat any of the part objects; and since this was the only thing that relied on `RadosStore`, we should be able to drop this dynamic_cast and TODO as well

Reply:
yes, removed as discussed